site/docs-md/guides/push-notifications-firebase.md

```diff
@@ -4,11 +4,12 @@ description: Learn how to get Firebase Cloud Messaging working on iOS and Androi
 url: /docs/guides/push-notifications-firebase
 contributors:
   - bryplano
+  - javebratt
 ---
 
```

Comment:
Under Required Dependencies section, can you add something to help avoid the issue where devs are using out of date firebase dependencies etc?  Because they get stale quickly, I'd like to avoid recommending "x version of node, y version of android.", unless it's a hard requirement like major versions, "iOS 13+" for example. Instead, how about something like "If you encounter issues like X, ensure that you're on the latest versions of Node, Android Studio, and Xcode".

Also, for one issue, the problem was that someone used `phonegap-plugin-push` which used FB messaging 2.x when latest is 4.x - any way to address that generically? like "Check that you're using the latest Firebase SDKs (list those that are used in this guide) as well as ensure that you're not using multiple copies of FB, that may be included in other plugins (especially Cordova ones like X, Y, Z (pg-plugin-push). 

If there's a bunch of details that would help others, we could add a Troubleshooting section at the very end of the guide instead, as well. up to you.

Reply:
Alright, I added a small note, don't want to give hard numbers as you say, but pushed people to use the latest **stable** version.